### DISCUSSION TITLE
Gradle Installation error

### DISCUSSION THREAD

**Discussion by YesShubham01** [oppia/oppia-android#4810] [Q&A (Installation)] [answered]
I am installing for the first time.. but for some reason, I am unable to run it. 
getting this error 
~"Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0."
Can anyone please guide me?

![Screenshot 2022-12-28 at 8 56 14 PM](https://user-images.githubusercontent.com/116451766/209834419-b7b6a233-f3e8-4715-aeff-e8cac7000eb6.png)

**Comment by MohitGupta121** [comment]
@YesShubham01 Which Android Studio version you are using?
Recommended version for oppia-android is - **Android Studio Bumblebee** 

I think you changed the gradle version, so please keep it same in `gradle-wrapper.properties` as `distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-bin.zip` and after this try Android Studio toolbar (Build) -> Clean Project -> Rebuild Project.


If it fixed by this answer please mark this as answer or if you find any other way that work for you please share and mark that as answer. So that this helps others in community.

**Reply by MohitGupta121** [reply]
Hey @YesShubham01 if your issue is resolved please mark this discussion is resolved so that it's helpful to others in the community.
And if you need help then also feel free to ask.

**Comment by BenHenning** [comment, marked as answer]
Unless I'm mistaken, it doesn't actually seem like there's an issue here. The build seems to have succeeded, so the message just appears to be a warning. We don't use Gradle 7.0, so this warning is fine to ignore.

**Comment by MohitGupta121** [comment]
@YesShubham01 I think with Ben's answer https://github.com/oppia/oppia-android/discussions/4810#discussioncomment-4640041 your issue is resolved please mark this discussion is resolved so that it's helpful to others in the community. And if you need help then also feel free to ask.
